official/copy_paste_metadata.lua

```diff
@@ -58,11 +58,8 @@ local publisher = ""
 local rights = ""
 local tags = {}
 
--- Tell gettext where to find the .mo file translating messages for a particular domain
-gettext.bindtextdomain("copy_paste_metadata",dt.configuration.config_dir.."/lua/locale/")
```

Comment:
change this line to `gettext.bindtextdomain("darktable", dt.configuration.locale_dir)`

Reply:
I may be missing something, but why is it better than removing the line? What did I broke?

Reply:
Actually nothing.  I just wasn't sure what Lua would inherit from darktable.  I tried it with and without the line and it works the same either way.  So, we can get rid of it.